account_credit_control/models/credit_control_line.py

```diff
@@ -247,6 +259,15 @@ def unlink(self):
                     _('You are not allowed to delete a credit control '
                       'line that is not in draft state.')
                 )
+            if line.policy_id.auto_process_lower_levels:
+                related_lines = line.get_related_lines().filtered(
+                    lambda l: l.id != line.id
+                )
+                if not related_lines:
```

Comment:
@qgroulard Maybe a helper function to reuse code that writes on related_lines ?

Reply:
I also think it would be nice to have a helper here since there is some code reused several times.

Reply:
Not relevant anymore, this part of code has been removed for simplification.